### ISSUE TITLE
[Feature Request]: Support for pressure sensitive buttons of the DualShock 2/3

### ISSUE THREAD

**Issue by brad86m** [PCSX2/pcsx2#6233]
### Description

I use a Dualshock 3 in combination with DsHidMini/BthPS3, and it is great for using the controller on PC with all its features. For some reason, though, PCSX2 no longer supports the anolog pressure sensitive input of these controllers.

### Reason

Currently, can't get the most from the games that make use of it.

### Examples

N/A

**Comment by refractionpcsx2** [comment]
It should do, I'm pretty sure I remember @CookiePLMonster implementing it.

Did you make sure to enable XInput in the controller settings (and maybe disable SDL) ?

![image](https://user-images.githubusercontent.com/6278726/170826270-6daaa46d-be18-45a4-bd96-31dd6881c2ce.png)

**Comment by brad86m** [comment]
Tried everything that should enable the ability to use analogue inputs within the emulator. It's literally no longer available within PCSX2. Before the removal of external plugins, it worked fine.

![Untitled](https://user-images.githubusercontent.com/84683672/170856584-addcc6d2-b6cb-4481-8bf8-afeb05dc68ea.png)

**Comment by CookiePLMonster** [comment]
You are supposed to use the SXS mode in tandem with DsHidMini's `xinput1_3.dll` in PCSX2 directory.

**Comment by brad86m** [comment]
Thanks for the reply. I didn't realize the xinput dll needed to be in PCSX2 folder. Sadly, it is still, not working for me. Buttons are still acting as digital.

**Comment by brad86m** [comment]
Wired isn't viable for me, but I appreciate the help. I require BthPS3 for the DualShock 3 to work as intended wirelessly, and it does just that perfectly with everything I have been using it for except PCSX2, sadly.

**Comment by jonasos3** [comment]
Are you guys able to get vibration to work when using pressure sensitive buttons (with DsHidMini)? The pressure sensitivity works great for me with the methods described in this post, but vibration seems to be broken.

**Comment by ja3h** [comment]
Yeah I got vibration working. For me when manually adding the motors only SDL ones are detected. I add those then edit PCSX2.ini to change `LargeMotor = SDL-0` to `LargeMotor = XInput-0`, then repeat the same for SmallMotor. 
As an aside the Left Analog always seems to flip the vertical inputs if I automatically bind the DS3 but again that's easily fixed in the ini.

**Comment by mirh** [comment]
@nefarius I feel like you should update your HID recommendations in light of this

**Comment by nefarius** [comment]
> @nefarius I feel like you should update your HID recommendations in light of this

Sorry my what? Wasn't paying attention to this in a while, could someone bring me up to speed pls.

**Comment by mirh** [comment]
I'm not really updated either tbh, but AFAICT from the conversation pcsx2 is now best served with SXS, not SDF.

**Comment by RedDevilus** [comment]
Yeah SDF is for WX PCSX2 which is still handy for compat sake but Qt PCSX2 targets SXS.

**Comment by nefarius** [comment]
I see, cool, thanks! Gonna update the recommendations on the docs site 👍

**Comment by mirh** [comment]
#4918
Wait a moment...
I think I'm having an aneurysm now about what works with what
